docusaurus/docs-internals/editor-controls/unit-controls/range.md

```diff
@@ -0,0 +1,272 @@
+  label: "Range"
+}
+```
+
+#### Devices examples
+
+It will be rendered on all devices. This value can be skipped because it is set to `"all"` by default.
+
+```js
+{
+  id: "interval",
+  type: "range",
+  devices: "all"
+}
+```
+
+Rendering will occur only on `desktop`.
+
+```js
+{
+  id: "interval",
+  type: "range",
+  devices: "desktop"
+}
+```
+
+The display is limited to responsive modes, specifically `tablet` and `mobile`.
+
+```js
+{
+  id: "interval",
+  type: "range",
+  devices: "responsive"
+}
+```
+
+#### Disabled examples
+
+Control will be disabled. Normally, here should be your dynamic condition.
+
+```js
+{
+  id: "interval",
+  type: "range",
+  disabled: true
+}
+```
+
+Control will be disabled when `videoType` variable will be `"custom"`.
+`getValue` is a getter function that allows us to retrieve the value of controls by their id.
+`"videoType"` is the id of the `"select"` control below.
+
+```js
+const getToolbarContols = ({ getValue }) => {
+  const videoType = getValue("videoType");
+
+  return [
+    {
+      id: "videoType",
+      type: "select",
+      choices: [
+        { title: "Youtube", value: "youtube" },
+        { title: "Custom", value: "custom" },
+      ],
+    },
+    {
+      id: "interval",
+      type: "range",
+      disabled: videoType === "custom",
+    },
+  ];
+};
+```
+
+#### Helper examples
+
+`content` serves as the tooltip's content and also determines whether the icon, which triggers the tooltip on hover, will be visible.<br/> If `content` is an empty string the icon will not display.
+
+```js
+{
+  id: "interval",
+  type: "range",
+  helper: {
+    content: "Helper"
+  }
+}
+```
+
+`position` specifies the position of the tooltip relative to the icon.
+
+```js
+{
+  id: "interval",
+  type: "range",
+  helper: {
+    content: "Helper",
+    position: "top-start"
+  }
+}
+```
+
+#### Config `min` example
+
+Defines the minimum value allowed on the range scale.
+
+```js
+{
+  id: "interval",
+  type: "range",
+  config: {
+    min: 10
+  }
+}
+```
+
+#### Config `max` example
+
+Defines the maximum value allowed on the range scale.
+
+```js
+{
+  id: "interval",
+  type: "range",
+  config: {
+    max: 50
+  }
+}
+```
+
+#### Config `step` example
+
+Specifies the incremental step between values on the range scale.
+
+```js
+{
+  id: "interval",
+  type: "range",
+  config: {
+    step: 10
+  }
+}
+```
+
+#### Config `updateRate` example
+
+Determines the frequency of value updates, typically in milliseconds.
+
+```js
+{
+  id: "interval",
+  type: "range",
+  config: {
+    updateRate: 100
+  }
+}
+```
+
+#### Config `unit` example
+
+Specifies the unit of measurement for the values displayed on the range scale.
+
+```js
+{
+  id: "interval",
+  type: "range",
+  config: {
+    unit: "%"
+  }
+}
+```
+
+#### Config `startLabel` example
+
+Specifies the label indicating the starting point of the range.
+
+```js
+{
+  id: "interval",
+  type: "range",
+  config: {
+    startLabel: 10
```

Comment:
why you writed this as number if in parameters you defined it as string

![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/72bb330a-56b3-493c-b487-941e383f577d)


check also the endLabel